cli/requirements.txt

```diff
@@ -0,0 +1,2 @@
+click
```

Comment:
Should we pin versions here?

Reply:
Do we have a general style guide on pinning repositories? I prefer not to, because I've had bad experience with contextual version conflicts being a pain, I can also set a large range? Or down to a specific version if that's what's desired.

Reply:
We don't have a guide yet (apart from analyses, where we recommend pinning versions for reproducibility). The disadvantage of not pinning versions here is that one day this might stop compiling for an unrelated commit, because of a breaking API change in one of the dependencies. As this is a standalone tool, would there be any drawback when pinning to whatever is latest right now?

Reply:
Pinned to:

```
google-auth ==1.25.0
click ==7.1.2
```